pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -779,8 +780,9 @@ func makeFirewallName(name string) string {
 	return fmt.Sprintf("k8s-fw-%s", name)
 }
 
-func makeFirewallDescription(ipAddress string) string {
-	return fmt.Sprintf("KubernetesAutoGenerated_OnlyAllowTrafficForDestinationIP_%s", ipAddress)
+func makeFirewallDescription(serviceName, ipAddress string) string {
+	return fmt.Sprintf(`KubernetesManaged_({"kubernetes.io/service-ip":"%s", "kubernetes.io/service-name":"%s"})`,
```

Comment:
I can't see @ArtfulCoder comments (maybe obliterated in a commit?) but he told me about them.  I actually disagree, I don't think we want or need the KubernetesManaged_() stuff.  We're creating the object, we own it.